include/cvc5/cvc5_proof_rule.h

```diff
@@ -2418,7 +2418,8 @@ enum ENUM(ProofRewriteRule) : uint32_t
    * **Sets - empty tester evaluation**
    *
    * .. math::
-   *   \mathit{sets.is\_empty}(as \ \mathit{set.empty} \ (\mathit{Set} \ T)) = \top
+   *   \mathit{sets.is\_empty}(as \ \mathit{set.empty} \ (\mathit{Set} \ T)) =
```

Comment:
```suggestion
   *   \mathit{sets.is\_empty}(as \ \mathit{set.empty} \ (\mathit{Set} \ \top)) =
```
Formatting potentially needed